hpcgap/lib/galois.gi

```diff
@@ -546,7 +546,7 @@ local f,n,i,sh,fu,ps,pps,ind,keineu,ba,bk,j,k,a,anz,pm,
   Info(InfoPerformance,2,"Using Transitive Groups Library");
   f:=arg[1];
   f:=f/LeadingCoefficient(f);
-  if not(IsIrreducible(f)) then
+  if not(IsIrreducibleRingElement(f)) then
```

Comment:
What is this change about? It is not mentioned in the commit message nor the PR description.

Reply:
`IsIrreducible` defined in `overload.g` and delegates to `IsIrreducibleRingElement`. Since the order in which definitions are read changes, I had to to this to allow `galois.gi` to be read before `overload.g`.